### ISSUE TITLE
Most foolproof way to install odgi?

### ISSUE THREAD

**Issue by anshumanmohan** [cucapra/pollen#127]
When doing up #126 I was reminded that the installations we provide for installing odgi are wanting. See [this](https://github.com/cucapra/pollen/tree/better-readme#odgi). We sorta point them to too many places: bioconda, installing from source (which has a sorta vague pointer to path-editing), and we finally walk through a hacky middle-ground option.

What do people actually use? I use a built-from-source copy, and I've done the path-modification thing to get Python bindings to work:
```
export PYTHONPATH=/scratch/anshuman/odgi-py/lib/python3.9/site-packages/
export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so.2
```
It's not that bad. Does any of us do differently? Thoughts? These instructions will now live in the bottom of the file and will be intended for the most serious users, those who want not just to use Pollen but to develop and extend it. I think we can give them one straight-line route, even if it's a little involved.

**Comment by sampsyo** [comment]
That route actually seems pretty good to me. I have never personally messed with Conda/Bioconda, and it seems like overkill for our purposes. Building from source in practice works pretty good for me, and to the extent that we need the Python bindings (hopefully we can move away from this, a la #116), extending those paths via environment variables seems like a good, simple way to get access to those.

If this approach works at least once for others (@priyasrikumar, do I recall that you maybe wanted to give this a shot?), maybe we should recommend this for now. And hopefully delete the env var stuff once we no longer need the Python bindings.

**Comment by priyasrikumar** [comment]
Hey! I gave this a shot this morning, but seem to be having some trouble getting OpenMP to be seen by CMake (similar to these posts: [1](https://stackoverflow.com/questions/72706342/openmp-mac-m1-gcc-and-libomp-not-working), [2](https://stackoverflow.com/questions/46414660/macos-cmake-and-openmp), [3](https://stackoverflow.com/questions/48825416/missing-openmp-c-flags-openmp-c-lib-names)). I've been troubleshooting but may need to try again later today after getting some other stuff out of the way. Either way, I think recommending a build from source in the README makes the most sense. There were some extra steps I had to take to fix errors that came up for me during the odgi installation; I have been keeping track of them + once I get odgi working I can update the README with some pointers!
